docs/GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+# Beheer 
+Ten behoeve van het beheer van de code is het noodzakelijk om afspraken te maken, zodat iedereen weet waar men aan toe is.
+
+## Gedachte Open Webconcept
+Binnen het Open Webconcept hanteren we de volgende uitgangspunten:
+* WordPress
+* Open architectuur:
+* Vrije marktwerking
+* Open samenwerking
+* Pragmatisch
+  Intentieverklaring op: https://OpenWebconcept.nl/intentieverklaring/
+
+## Open Webconcept Bouwblokken
+Binnen het Open Webconcept delen we functionaliteiten door middel van bouwblokken, hiervoor gelden de volgende uitgangspunten:
+* Te gebruiken in WordPress
+* Commonground gedachte
+* Delen met anderen
+* Helpen met fouten
+* Het is een Datadienst of een Toepassing
+
+## Behoefte Bouwblokken
+Voor deze bouwblokkken kunnen we dan ook bepaalde eisen ophalen:
+* Wordpress: opensource
+* Toegankelijk voor nieuwe nieuwe partijen  die willen participeren
+* Pragmatische insteek, alleen formeel wanneer het nodig is
+* Ook een reeële manier om weer te verlaten
+* Garanties ten opzichte van elkaar
+* Bouwblokken hebben altijd een [publiccode.yaml](https://yml.publiccode.tools/)
+* Bouwblokken staan bij voorkeur op [github.com/openwebconcept](https://github.com/openwebconcept)
+* Losse componenent:
+    * Code deelbaar
+    * Zelfde licentie
+    * 2e lijns support op de code (bugs/changes)
+    * Eén jaar aangeven dat support vervalt
+    * Eén plek waar alle bouwblokken te vinden zijn
+    * Signaal wanneer er een nieuwe versie is
+
+## Invulling Bouwblokken
+Per bouwblok:
+* Te vinden op https://github.com/OpenWebconcept/
+* Licentie is EUPL
+* Eén repository met die bloknaam
```

Comment:
Bedoel je per Bouwblok een losse repo in de vorm van een plug-in? In dat geval zou ik eerder kiezen voor 1 plug-in die alle blokken huisvest.

Reply:
@mvdhoek1 Ik begrijp je niet compleet: Wanneer je een bouwblok installeert wil je volgens mij alleen installeren wat je nodig hebt. Wat zou het argument zijn om meerdere bouwblokken in 1 plugin te huisvesten?

Reply:
De blokken waar we naar refereren zijn, neem ik aan, Gutenberg blokken toch? Deze kunnen ook ServerSide Rendered blokken zijn waardoor je deze in een plug-in dient te plaatsen. Als we voor ieder blok een plug-in moeten maken zou ik het logischer vinden op 1 plug-in te maken die alle blokken huisvest.

Als het niet over Gutenberg blokken is mijn reactie niet relevant.

Reply:
Ik denk dat een (bouw)blok refereert naar het volledige component. Dus een bouwblok is de base OpenPDC of de OpenPUB.